queries.sql

```diff
@@ -117,3 +117,55 @@ SELECT animals.name FROM animals JOIN owners ON animals.owner_id = owners.id WHE
 
 /* Who owns the most animals? */
 SELECT owners.full_name FROM owners JOIN animals ON owners.id = animals.owner_id GROUP BY owners.full_name ORDER BY COUNT(*) DESC LIMIT 1;
+
+/*Who was the last animal seen by William Tatcher?*/
+SELECT animals.name, visits.date_of_visit from animals
+JOIN visits ON visits.animals_id = animals.id
+JOIN vets ON vets.id =visits.vets_id
+WHERE vets.name = 'William Tatcher'
+ORDER BY visits.date_of_visit DESC LIMIT 1;
+/*How many different animals did Stephanie Mendez see?*/
+SELECT COUNT(*) FROM animals
+JOIN visits ON visits.animals_id = animals.id
+JOIN vets ON vets.id = visits.vets_id
+WHERE vets.name = 'Stephanie Mendez';
+/*List all vets and their specialties, including vets with no specialties./*
+SELECT vets.name, species.name FROM vets
+FULL JOIN specializations ON specializations.vets_id = vets.id
+FULL JOIN species ON species.id = specializations.species_id;
```

Comment:
- [ ] It looks like you have commented out these query commands here while they were answering a particular question. I would suggest that you uncomment them and refine them to be correct 👍 
HINT 💡  I think it was due to the comment on line 132 which you have misplaced. Check again well because you may have put the asterisk (*) after the bask slash (/) while it is supposed to come before it 😉